src/ui/components/OgreTimeline.tsx

```diff
@@ -34,7 +34,7 @@ const Spans = ({
   const { endTime } = useSelector(getZoomRegion)!;
```

Comment:
Lets rename this component `ProtocolTimeline` since we'll show all the data we have from the protocol